index.test.js

```diff
@@ -0,0 +1,111 @@
+const request = require('supertest');
+const server = require('./index');
+
+describe('index.js', () => {
+
+  describe('GET /games', () => {
+
+    it('should return status code 200', async () => {
+
+      const response = await request(server).get('/games');
+
+      expect(response.status).toBe(200);
+
+    });
+
+    it('should return a list of games', async () => {
+
+      const response = await request(server).get('/games');
+
+      expect(Array.isArray(response.body)).toBe(true);
+
+    });
+
+    it('should return JSON', async () => {
+
+      const response = await request(server).get('/games');
```

Comment:
Since you make this same request through these tests, why not use a `beforeAll`? For example:
```javascript
describe('GET /games', () => {
  let response;
  beforeAll(async () => {
    response = await request(server).get('/games');
  });

  it('should return status code 200', () => {
    expect(response.status).toBe(200);
  });

  it('should return a list of games', () => {
    expect(Array.isArray(response.body)).toBe(true);
  });

  // ...
```